lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -218,13 +218,13 @@
             this.backend = this.backends[obj.backendName] || obj.backend;
             this.captureSoundPath =  obj.captureSoundPath || "";
 
+            _.extend( this.backend, Backbone.Events );
```

Comment:
Reviewers: I had to move this line up here because `this.backend.initialize(...)` results in the `backend` object (which his not a Backbone object) to call a method inherited from `Backbone.Events`.